projects/dex-ui/src/wells/useBeanstalkSiloWhitelist.ts

```diff
@@ -1,24 +1,50 @@
 import { useMemo } from "react";
 import { Well } from "@beanstalk/sdk/Wells";
 
+const BEANETH = "0xbea0e11282e2bb5893bece110cf199501e872bad";
```

Comment:
Would be better to extract hard coded addresses to a central, easily to find, file like 'addresses.tx' or so..